state-chain/runtime/src/lib.rs

```diff
@@ -788,6 +788,7 @@ impl pallet_cf_threshold_signature::Config<Instance16> for Runtime {
 	type ThresholdCallable = RuntimeCall;
 	type ThresholdSignerNomination = chainflip::RandomSignerNomination;
 	type TargetChainCrypto = EvmCrypto;
+	type TargetChain = Arbitrum;
```

Comment:
What about Ehthereum?

Reply:
We don't instantiate `pallet_cf_threshold_signature::Config` for `Instance1` anywhere.

Reply:
Exactly: it's instantiated for the EVM, it's neither Arbitrum nor Ethereum.